packages/orbit-components/src/Heading/Heading.stories.tsx

```diff
@@ -1,442 +1,244 @@
+      },
+    },
+    align: {
+      options: Object.values(ALIGN),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const TitleDisplaySubtitle = ({ customTitle, as }) => {
-  return (
-    <Heading type="displaySubtitle" as={as}>
-      {customTitle}
-    </Heading>
-  );
-};
-
-TitleDisplaySubtitle.story = {
-  name: "Title displaySubtitle",
+export default meta;
+type Story = StoryObj<HeadingPropsAndCustomArgs>;
 
+export const Default: Story = {
   parameters: {
-    info: "Headings are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
-  },
-};
-
-TitleDisplaySubtitle.args = {
-  customTitle: "Orbit design system",
-  as: ELEMENT_OPTIONS.H1,
-};
-
-TitleDisplaySubtitle.argTypes = {
-  as: {
-    options: Object.values(ELEMENT_OPTIONS),
-    control: {
-      type: "select",
+    controls: {
+      exclude: ["type", "as", "inverted", "align", "spaceAfter"],
     },
   },
 };
 
-export const Title0 = ({ customTitle, as }) => {
-  return (
-    <Heading type="title0" as={as}>
-      {customTitle}
-    </Heading>
-  );
-};
-
-Title0.story = {
-  parameters: {
-    info: "Headings are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+export const TitleDisplay: Story = {
+  args: {
+    as: ELEMENT_OPTIONS.H1,
+    type: TYPE_OPTIONS.DISPLAY,
   },
-};
-
-Title0.args = {
-  customTitle: "Orbit design system",
-  as: ELEMENT_OPTIONS.H1,
-};
 
-Title0.argTypes = {
-  as: {
-    options: Object.values(ELEMENT_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["inverted"],
     },
   },
 };
 
-export const Title1 = ({ customTitle, as }) => {
-  return (
-    <Heading type="title1" as={as}>
-      {customTitle}
-    </Heading>
-  );
-};
-
-Title1.story = {
-  parameters: {
-    info: "Headings are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+export const TitleDisplaySubtitle: Story = {
+  args: {
+    as: ELEMENT_OPTIONS.H1,
+    type: TYPE_OPTIONS.DISPLAYSUBTITLE,
   },
-};
-
-Title1.args = {
-  customTitle: "Orbit design system",
-  as: ELEMENT_OPTIONS.H1,
-};
 
-Title1.argTypes = {
-  as: {
-    options: Object.values(ELEMENT_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["inverted"],
     },
   },
 };
 
-export const Title2 = ({ customTitle, as }) => {
-  return (
-    <Heading type="title2" as={as}>
-      {customTitle}
-    </Heading>
-  );
-};
-
-Title2.story = {
-  parameters: {
-    info: "Headings are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+export const Title0: Story = {
```

Comment:
I am thinking if it does make sense to have 7 almost same stories. I am facing same "issue" in `Box` component - same stories with different property name in Story name. What do you think of "squashing" them into 1 story? I don't know what's the best practice, tbh. 🥲

Reply:
I thought about this and it makes sense but then I re-read [this comment](https://github.com/kiwicom/orbit/pull/4437#issuecomment-2276154525) where we discussed it and considering this, I wouldn't squash them to one story. On one hand yes, it's almost the same 7 stories, but if I imagine wanting to use simple Heading component, when I find it in SB, I can just go over all different heading sizes without changing two props. I think it's the same with Badge, there we could also squash them into one story 🤔 And it should be direct migration so I would just keep it like this, WDYT?

Reply:
yeah, it makes sense to have a specific story for a specific heading type. 🤔 I was comparing it with "my" Box component as I squash multiple stories, but I guess that it makes more sense for Box component. 😄